### DISCUSSION TITLE
Crash when right clicking the start button

### DISCUSSION THREAD

**Discussion by mattcoz** [valinet/ExplorerPatcher#2182] [General]
Everything seems to be working fine except when I right click on the start button. I get the spinning cursor and then explorer crashes and restarts. I'm using the W10 taskbar and the W10 start menu. I enabled the console but nothing is jumping out at me as a clue. Let me know if you want any more information.

ExplorerPatcher 22621.2361.58.1
Windows 22621.2361

**Comment by mattcoz** [comment]
Oh wow, I right clicked the system tray overflow icon and it really got screwed up. It crashed like before, but didn't restart. I went into EP properties, clicked restart file explorer, which worked but all my taskbar icons are now really wide and the system tray has a border around it. Right clicking on the taskbar brings up the context menu but there's no text.

**Comment by Amrsatrio** [comment]
Works just fine on my laptop with the exact build as yours. English (US) version but with the display language on my user account only set to Japanese. I also use OldNewExplorer.

Does your console (EP Properties > Advanced > Enable console) have the same output as mine?
Any tweak software other than EP that you're using? (e.g. OldNewExplorer, TranslucentTB)

I suggest to send screenshots so we can understand better what's happening.

![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/922cd318-93a2-4fe6-a8c5-56f9846db626)

**Reply by mattcoz** [reply]
Hmm, my console is missing `symbols_PTRS.twinui_pcshell_PTRS[0]` through `symbols_PTRS.twinui_pcshell_PTRS[5]`, and I see this later on: `[Symbols] Failure in reading symbols for "C:\WINDOWS\system32\twinui.pcshell.dll".`

I'm using 7+ Taskbar Tweaker, but there's no change if I disable that. I tried uninstalling and reinstalling EP, reverting to default settings, but no change. Here is what my taskbar looks like now:

![image](https://github.com/valinet/ExplorerPatcher/assets/335855/157f0774-b9f5-4000-a958-bfff1d1625bb)

**Reply by Amrsatrio** [reply]
Interesting. Don't tell me that I have to make EP scan the stored DLL file instead of the loaded one in memory (which was modified by 7TT before EP scans it) 😬

**Reply by mattcoz** [reply]
Hmm, I've never had issues with 7tt before. I've tried uninstalling EP and reinstalling without 7tt, but there is no change. I don't know if that should have done anything or not. Is there anything you want me to try?

**Reply by Amrsatrio** [reply]
Are you sure there are no other DLLs loaded into Explorer though? Because the DLLs seem to be hooking the functions necessary for Win+X and skinning the menus before EP did. Currently EP expects the original data (unhooked, fresh from your disk) in order to succeed. I will try to fix this issue later this week. In the meantime, not sure if this helps, please send your twinui.pcshell.dll from system32 here with pdf extension.

**Reply by mattcoz** [reply]
I can't think of anything, certainly nothing that would have been added recently. Here's the DLL:
[twinui.pcshell.dll.pdf](https://github.com/valinet/ExplorerPatcher/files/12728489/twinui.pcshell.dll.pdf)

**Reply by Amrsatrio** [reply]
Another user of 22621.2361 with 7+TT experienced similar issues to yours. https://github.com/valinet/ExplorerPatcher/discussions/2194

**Comment by Kalenias** [comment]
I guess the update has fixed that bug, it happened to me but it doesnt happen anymore, If you have Start11 it may be the problem

**Reply by mattcoz** [reply]
I'm on the latest update though.

**Comment by pyrates999** [comment]
Insider builds are not supported: https://www.elevenforum.com/t/kb5030310-windows-11-insider-release-preview-build-22621-2361-22h2.17969/

**Reply by mattcoz** [reply]
It's the release preview build, not a beta build, and it's listed in the "tested on builds" for this release of EP. One of the points of release preview builds is so developers can test their software for compatibility before it is released to the masses.

**Reply by pyrates999** [reply]
Unless you're a developer, you shouldn't be installing release previews.

**Reply by mattcoz** [reply]
Good thing I am a developer then.

**Reply by pyrates999** [reply]
thanks for letting everyone know then.  Are you planning to contribute to EP?

**Comment by Amrsatrio** [comment]
Hi, I tried to make a fix to your issue if 7+TT is the one to blame. Please download the latest prerelease and see if the problem still persists. Also please make sure to be connected to the Internet when installing EP so that these files get downloaded properly for the Win10 start menu to work:
```
C:\Windows\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy\AppResolverLegacy.dll
C:\Windows\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy\StartTileDataLegacy.dll
C:\Windows\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy\Windows.UI.ShellCommon.pri
C:\Windows\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy\en-US\StartTileDataLegacy.dll.mui
C:\Windows\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy\pris2\Windows.UI.ShellCommon.en-US.pri
```

**Reply by mattcoz** [reply]
That fixed the Win+X menu, nice work! I'm still having issues with the display of taskbar items and the system tray, but I'm thinking something got screwed up somewhere and it may have nothing to do with EP.

**Reply by pyrates999** [reply]
Check using the console in explorer patcher.  Open up the properties application for explorer patcher, go to advanced, and click on enable console.  If it shows it can't download symbols, your version of windows is not supported.
